CommandLine/libEGM/gmx.cpp

```diff
@@ -338,6 +331,25 @@ void PackRes(std::string &dir, int id, pugi::xml_node &node, google::protobuf::M
       if (gmxName == "GMX_DEPRECATED")
         continue;
 
+      if (gmxName == "eventtype") {
```

Comment:
I haven't figured out why this breaks things, yet, but this is clearly the product of a merge error. This logic already appears below. We have a duplicate handler for "action," as well. Mind deleting those and retrying? It shouldn't fix  this problem, but it should help us narrow it down.

To me, it's important to figure out why this change breaks it, because there may be a root cause I'm not seeing and will therefore trip again.

Speaking of, can we get an integration test that actually fails on this PR?

Reply:
Everything should fail on this pr. At least any test harness calls